playbook/app/pb_kits/playbook/pb_circle_icon_button/_circle_icon_button.tsx

```diff
@@ -13,21 +13,21 @@ import Button from '../pb_button/_button'
 import Icon from '../pb_icon/_icon'
 
 type CircleIconButtonProps = {
-  aria?: object,
+  aria?: { [key: string]: string },
   className?: string,
   dark?: boolean,
-  data?: object,
+  data?: { [key: string]: string },
   disabled?: boolean,
   icon: string,
   id?: string,
   link?: string,
-  onClick?: Callback,
+  onClick?: EventHandler,
```

Comment:
Was there a reason for separating this out onto line #7, instead of just keeping it all on one line?